about.html

```diff
@@ -0,0 +1,175 @@
+          </p>
+          <div class="about-question">
+            <span class="about-hero-question">
+              For more details reach out to us at
+            </span>
+            <span class="about-hero-email">lusakafashionweek2022@gmail.com</span>
+          </div>
+        </div>
+      </section>
+      <section id="the-logo">
+        <div class="the-logo-section desktop-padding">
+          <h2 class="the-logo-title the-underline">THE SHOWROOMS</h2>
+          <p class="d-logo-description">
+            Lusaka show rooms provides a unique opportunity for emerging Zambian 
+            designers to promote their brands to an international audience of buyers and will be
+             at playhouse in Lusaka from 6th June to 11th June 2022 with selected designers  
+          </p>
+          <p class="m-logo-description">
+            The main runway display and its stunning view from the event.
+          </p>
+          <div class="the-logo-container">
+            <img src="images/main1.jpg" alt="Logo of the training">
+          </div>
+        </div>
+      </section>
+      <section id="past-training">
+        <div class="past-training desktop-padding">
+          <h2 class="past-training-title the-underline">Previous Events</h2>
+          <p>Looking back at the last two Lusaka fashion events</p>
+          <div class="past-training-container">
+            <div class="past-training-item aau">
+              <h3>2021</h3>
+              <p>Lusaka Fashion Week 2021</p>
+            </div>
+            <div class="past-training-item ju">
+              <h3>2020</h3>
+              <p>The Lusaka Fashion Week  2020</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="about-partners">
+        <h2 class="partners-section-title the-underline">Partners</h2>
+        <div class="partners-container">
+          <img
+            class="partner-image"
+            src="images/social/mtn.svg"
+            alt="mtn"
+          />
+          <img
+            class="partner-image"
+            src="images/social/Louis_Vuitton.svg"
+            alt="Louis_Vuitton"
+          />
+          <img
+            class="partner-image"
+            src="images/social/New_Balance.svg"
+            alt="Google"
+          />
+          <img
+            class="partner-image"
+            src="images/social/toyota.svg"
+            alt="toyota"
+          />
+          <img
+            class="partner-image"
+            src="images/social/airtel.svg"
+            alt="airtel"
+          />
+          <img
+            class="partner-image"
+            src="images/social/shoprite.svg"
+            alt="shoprite"
+          />
+        </div>
+      </section>
+    </main>
+    <footer class="about-footer">
+      <a href="/">
+        <img class="black-footer-logo" src="images/log02.svg" alt="Logo" />
+        
+      </a>
+      <p>2022 Lusaka Fashion Week. All rights reserved.</p>
+    </footer>
```

Comment:
- [ ] Hi, this section requires that you have a logo aligned to the left and text aligned to the right currently you only have text. please try to match the design by implementing this feature.

Reference Design | Your Design
-|-
![image](https://user-images.githubusercontent.com/18730106/158032533-ec2bf1fe-16b3-44be-bc33-83ea536ab6e2.png)|![image](https://user-images.githubusercontent.com/18730106/158032545-22bb9809-9d37-46d5-a892-5e0962c5dd50.png)